airbyte-cdk/java/airbyte-cdk/build.gradle

```diff
@@ -10,25 +10,13 @@ allprojects {
 
     group 'io.airbyte.cdk'
 
-    def artifactBaseName = 'airbyte-cdk-' + project.name
-    // E.g. airbyte-cdk-core, airbyte-cdk-db-sources, airbyte-cdk-db-destinations, etc.
-
     project.version = cdkVersion
 
```

Comment:
out of scope, was trying to get sources published (helps when debugging line numbers instead of intellij decompiled class). I couldn't get this block working here 
```
java {
    withSourcesJar()
}
```

Reply:
Good catch. Yes, let's publish sources too. I pushed a commit which fixes this. It's pretty low touch so I feel good about including it here.